### PR TITLE
Updater: Show error when trying to launch directly

### PR DESCRIPTION
**Pull request by spycrab**

### CODE REVIEW

File: Source/Core/Updater/Main.cpp [hunk 1]
```diff
@@ -670,6 +670,15 @@ void FatalError(const std::string& message)
 
 int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PWSTR pCmdLine, int nCmdShow)
 {
+  if (lstrlenW(pCmdLine) == 0)
+  {
+    MessageBox(nullptr,
+               L"This updater is not meant to be launched directly. Configure Auto-Update in "
+               "Dolphins Settings instead.",
```
Comment (MayImilae) [inline Source/Core/Updater/Main.cpp]: I believe that should be "Dolphin's"? Also, settings should probably not be capitalized.

File: Source/Core/Updater/Main.cpp [hunk 2]
```diff
@@ -670,6 +670,15 @@ void FatalError(const std::string& message)
 
 int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PWSTR pCmdLine, int nCmdShow)
 {
+  if (lstrlenW(pCmdLine) == 0)
+  {
+    MessageBox(nullptr,
+               L"This updater is not meant to be launched directly. Configure Auto-Update in "
+               "Dolphin's settings instead.",
```
Comment (BhaaLseN) [inline Source/Core/Updater/Main.cpp]: If you keep this line broken up, it should have the `L` prefix as well.

Comment (spycrab) [reply Source/Core/Updater/Main.cpp]: No need to. It will still be a proper ``wchar_t*``.

Comment (BhaaLseN) [reply Source/Core/Updater/Main.cpp]: You're concatenating a `wchar_t*` with a `char*` there. Intentional or not, it should have the prefix (even if you'll never go and add characters that don't fit into a `char` later).

Comment (spycrab) [reply Source/Core/Updater/Main.cpp]: Not according to Visual Studio and [cppreference](http://en.cppreference.com/w/cpp/language/string_literal):

>  If one of the strings has an encoding prefix and the other doesn't, the one that doesn't will be considered to have the same encoding prefix as the other. 

 Also: </img>.

Comment (BhaaLseN) [reply Source/Core/Updater/Main.cpp]: But I want my shed painted green :^)
I'll just take cppreference's word for that; back in the day (which is already 15 years past) it had to be the same. VS docs don't really mention it at all, so I assumed it still is.

Comment (spycrab) [reply Source/Core/Updater/Main.cpp]: The segment I was quoting has a ``(since C++11)`` note so that's not a surprise I guess.

**Pull request merged**